networkit/cpp/graph/Graph.cpp

```diff
@@ -600,22 +600,21 @@ void erase(node u, index idx, std::vector<std::vector<T>> &vec) {
     vec[u].pop_back();
 }
 
-void Graph::removeEdge(node u, node v, bool maintainSortedEdges, bool maintainCompactEdgeIDs) {
+void Graph::removeEdge(node u, node v) {
     assert(u < z);
     assert(exists[u]);
     assert(v < z);
     assert(exists[v]);
 
-    if (maintainCompactEdgeIDs && !edgesIndexed) {
-        throw std::runtime_error(
-            "Edges have to be indexed if maintainCompactEdgeIDs is set to true");
+    if (maintainCompactEdges && !edgesIndexed) {
+        throw std::runtime_error("Edges have to be indexed if maintainCompactEdges is set to true");
     }
 
     index vi = indexInOutEdgeArray(u, v);
     index ui = indexInInEdgeArray(v, u);
 
     node deletedID;
-    if (maintainCompactEdgeIDs) {
+    if (maintainCompactEdges) {
         deletedID = outEdgeIds[u][vi];
     }
```

Comment:
To fix the clang-tidy error, you can probably move `deletedID = outEdgeIds[u][vi];` below so that you define it only if you use it.